index.html

```diff
@@ -318,41 +318,30 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li><a>Patronizing</a> language or behavior:
+            <ul>
+              <li>Intentionally or unintentionally making assumptions about the skills or knowledge of others, such as using language that implies the audience is uninformed on a topic (e.g. interjections like "I can't believe you don't know about [topic]").
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled due to their characteristics (e.g., “So easy your grandmother could do it”, which implies an older woman might not be technically competent).
+              </li>
+              <li>Interrupting or repeatedly commenting in conversations with unneccessary clarifications or comments on audience behaviour (e.g. "I don't think you understood my previous comment...", grammar or language corrections that were not invited).</li>
```

Comment:
1. This is more an example of boorishness or rudeness than of patronizing behavior.  I don't think it's a good example to include here.

2. "I don't think you understood my previous comment" seems to me like a completely appropriate statement for someone to make if that person is feeling not understood, so if it is offered as an example of patronizing behavior I think it needs context to clarify why.

Reply:
I would be strongly opposed to removing this.  This is very very common in practice, and it needs to be called out here.  It is not just boorish or rude; particularly when repeated, it encourages bad conduct.